### PR TITLE
build: Update changesets and add release notes

### PR DESCRIPTION
**Pull request by tylerbutler**
Updates the changesets for clarity and adds a release notes file that will be pasted into the GitHub release.

### CODE REVIEW

File: .changeset/red-llamas-spit.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-"@fluidframework/container-utils": major
+"@fluidframework/telemetry-utils": major
```
Comment (alexvy86) [inline .changeset/red-llamas-spit.md:2]: ?

**Review by alexvy86 — COMMENTED** [review]
One thing that threw me off below. And I don't see the new release notes file mentioned in the description?

File: .changeset/red-llamas-spit.md (hunk 1 above)
Comment (tylerbutler) [reply .changeset/red-llamas-spit.md:2]: You can't have a changeset that refers to a package that doesn't exist.

Comment (tylerbutler) [reply .changeset/red-llamas-spit.md:2]: Since most of the functions went to telemetry-tuils, I used that package.

Comment (alexvy86) [reply .changeset/red-llamas-spit.md:2]: Hmm leaves me wondering how do we announce "this package is now completely gone and there are no replacements for its exports" if we ever need to. And in fact I think it kind of applies here? The things that moved over, I imagine might not be intended for use by external consumers, and this makes it feel like if they were using them, they should now just use them from telemetry-utils. Disclaimer: I didn't go check _what_ moved, so might be wrong on that.

Comment (tylerbutler) [reply .changeset/red-llamas-spit.md:2]: > leaves me wondering how do we announce "this package is now completely gone and there are no replacements for its exports" if we ever need to

I'm not worried about that. In that case we don't really need to announce the package is gone. The last version of the package published will have all the APIs deprecated (e.g. we just did this with common-utils), and once deleted, there's just never a new version published. That's basically what happened here - we shipped deprecations in 6.2.0 (I assume we announced the deprecations in the release notes then), and that's the "final" version of that package.

In other words, the decision of replacement APIs and announcing them really needs to be made when deprecating them, not when removing them.

**Comment by tylerbutler** [comment]
> One thing that threw me off below. And I don't see the new release notes file mentioned in the description?

Sorry about that - they should be there now.

File: .changeset/sour-shrimps-start.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 "@fluid-internal/client-utils": minor
 ---
 
-Internal buffer encoding helpers now require 'utf8', 'utf-8', or 'base64'
+client-utils: Internal buffer encoding helpers now require 'utf8', 'utf-8', or 'base64'
```
Comment (alexvy86) [inline .changeset/sour-shrimps-start.md:5]: Curious: did you manually remove this one from the release notes file? I agree it's not relevant there, this being an internal package, just wondering how come it didn't show up.

File: RELEASE_NOTES/2.0.0-internal.7.0.0.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+Subsequently, various `entryPoint` and `getEntryPoint()` endpoints have become required. Please see [containerRuntime.ts](https://github.com/microsoft/FluidFramework/tree/main/packages/runtime/container-runtime/src/containerRuntime.ts) for example implementations of these APIs.
+
+For more details, see [Removing-IFluidRouter.md](https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/Removing-IFluidRouter.md)
+
+## container-runtime: Removing some deprecated and likely unused ContainerRuntime APIs
+
+-   `IGCRuntimeOptions.sweepAllowed`
+-   `ContainerRuntime.reSubmitFn`
+
+## container-runtime: initializeEntryPoint changed to provideEntryPoint
```
Comment (alexvy86) [inline RELEASE_NOTES/2.0.0-internal.7.0.0.md:96]: Nit: move this above the one that talks about the new name, further up?

File: .changeset/early-cobras-juggle.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 "@fluidframework/request-handler": major
```
Comment (markfields) [inline .changeset/early-cobras-juggle.md:5]: Could/should be `minor`, right?

File: RELEASE_NOTES/2.0.0-internal.7.0.0.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+The naming of `initializeEntryPoint` has been changed to `provideEntryPoint`. Please change the property name in relevant calls to `ContainerRuntime.loadRuntime(...)`.
+
+## container-definitions: IContainer's and IDataStore's IFluidRouter capabilities are deprecated
```
Comment (alexvy86) [inline RELEASE_NOTES/2.0.0-internal.7.0.0.md:100]: And move down
```suggestion
## DEPRECATED: container-definitions: IContainer's and IDataStore's IFluidRouter capabilities are deprecated
```

**Review by alexvy86 — APPROVED** [review]

File: .changeset/early-news-kick.md [hunk 6]
```diff
@@ -7,10 +7,20 @@
 "@fluidframework/odsp-driver-definitions": major
 ---
 
-Handle DriverErrorType.outOfStorage error from driver and load the container in readonly mode.
+container-loader: Load container in readonly mode when driver throws DriverErrorType.outOfStorage
```
Comment (markfields) [inline .changeset/early-news-kick.md:10]:
```suggestion
odsp-driver: Load container in readonly mode when driver throws DriverErrorType.outOfStorage
```

File: .changeset/early-cobras-juggle.md (hunk 4 above)
Comment (markfields) [reply .changeset/early-cobras-juggle.md:5]: And same for other changesets throughout that are just deprecations.

File: .changeset/green-forks-invite.md [hunk 7]
```diff
@@ -6,6 +6,6 @@
 "@fluidframework/routerlicious-driver": major
 ---
 
-"Removing dead concepts around blob aggregation"
+Blob aggregation: remove dead concepts and code
```
Comment (markfields) [inline .changeset/green-forks-invite.md:9]:
```suggestion
routerlicious: Blob aggregation - remove dead concepts and code
```

File: .changeset/early-cobras-juggle.md (hunk 4 above)
Comment (tylerbutler) [reply .changeset/early-cobras-juggle.md:5]: Could be, yeah. Today we base the major/minor on the target branch in the tools, so unless people change it themselves or make the changeset manually, all next-branch changes will be tagged major.

We don't do much with the info except group stuff in the per-package changelogs, and we only use it there because we inherit that functionality from the toolchain - we did not implement it. That could definitely change in the future though.

File: .changeset/loud-lions-worry.md [hunk 8]
```diff
@@ -2,6 +2,6 @@
 "@fluidframework/container-runtime": major
 ---
 
-initializeEntryPoint changed to provideEntryPoint
+container-runtime: initializeEntryPoint changed to provideEntryPoint
```
Comment (markfields) [inline .changeset/loud-lions-worry.md:5]:
```suggestion
container-runtime: initializeEntryPoint renamed to provideEntryPoint
```

File: .changeset/sour-shrimps-start.md (hunk 2 above)
Comment (tylerbutler) [reply .changeset/sour-shrimps-start.md:5]: I think this was a copy/paste fail when I reordered the markdown.

Comment (tylerbutler) [reply .changeset/sour-shrimps-start.md:5]: But not going to worry about fixing it since as you noted it's an internal package. The note will still make it into the client-utils changelog.

File: .changeset/green-forks-invite.md (hunk 7 above)
Comment (tylerbutler) [reply .changeset/green-forks-invite.md:9]:
```suggestion
routerlicious-driver: remove dead blob aggregation concepts and code
```

File: .changeset/modern-worlds-fry.md [hunk 9]
```diff
@@ -16,7 +16,7 @@
 "@fluidframework/test-utils": major
 ---
 
-provideEntryPoint is required
+container-runtime: provideEntryPoint is required
```
Comment (markfields) [inline .changeset/modern-worlds-fry.md:19]: I'd say `test-utils` here - yes it's about container-runtime functionality, but this changeset is about constructors on test util classes (I believe)

Comment (tylerbutler) [reply .changeset/modern-worlds-fry.md:19]:
```suggestion
test-utils: provideEntryPoint is required
```

File: .changeset/red-llamas-spit.md (hunk 1 above)
Comment (markfields) [reply .changeset/red-llamas-spit.md:2]: Maybe put in the body another sentence or phrase "Many of them were moved to telemetry-utils" or whatever

File: .changeset/seven-ghosts-try.md [hunk 10]
```diff
@@ -3,6 +3,6 @@
 "@fluid-experimental/tree": major
```
Comment (markfields) [inline .changeset/seven-ghosts-try.md:3]: Why is it this?

File: .changeset/upset-paths-stand.md [hunk 11]
```diff
@@ -2,6 +2,6 @@
 "@fluidframework/container-loader": major
 ---
 
-Container caching in the Loader is removed
+container-loader: Container caching in the Loader is removed
```
Comment (markfields) [inline .changeset/upset-paths-stand.md:5]: @alexvy86 - Can you follow up with stakeholders on FF/Loop here to make sure this won't end up blocking the bump?  It sounded like the migration was a little disjointed based on the firedrill that happened during the last major bump.

File: .changeset/red-llamas-spit.md (hunk 1 above)
Comment (tylerbutler) [reply .changeset/red-llamas-spit.md:2]: I think maybe you guys are missing the whole changeset content, which is:

> All members of the **@fluidframework/container-utils** package have been deprecated and the package is now removed.
> 
> Migration by API member:
> -   `ClientSessionExpiredError` (deprecated in `2.0.0-internal.6.2.0`): No replacement API offered.
> -   `DataCorruptionError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `DataProcessingError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `DeltaManagerProxyBase` (deprecated in `2.0.0-internal.6.1.0`): No replacement API offered.
> -   `extractSafePropertiesFromMessage` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `GenericError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `ThrottlingWarning` (deprecated in `2.0.0-internal.6.2.0`): No replacement API offered.
> -   `UsageError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.

I think that's pretty thorough.

File: RELEASE_NOTES/2.0.0-internal.7.0.0.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+-   `DataCorruptionError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
+-   `DataProcessingError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
+-   `DeltaManagerProxyBase` (deprecated in `2.0.0-internal.6.1.0`): No replacement API offered.
+-   `extractSafePropertiesFromMessage` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
+-   `GenericError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
+-   `ThrottlingWarning` (deprecated in `2.0.0-internal.6.2.0`): No replacement API offered.
+-   `UsageError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
+
+## DEPRECATED: resolveHandle and IFluidHandleContext deprecated on IContainerRuntime
```
Comment (markfields) [inline RELEASE_NOTES/2.0.0-internal.7.0.0.md:178]: Group these deprecations under a sub-heading?

**Review by markfields — APPROVED** [review]

File: .changeset/seven-ghosts-try.md (hunk 10 above)
Comment (tylerbutler) [reply .changeset/seven-ghosts-try.md:3]: The original PR made changes to tree: https://github.com/microsoft/FluidFramework/pull/17210/files

Comment (markfields) [reply .changeset/seven-ghosts-try.md:3]: Ok yeah I see that.  I think that original changeset should have higlighted container-loader not tree.  This is the core of that change as far as I can see:  https://github.com/microsoft/FluidFramework/pull/17210/files#diff-c94f4bb2aaac11f9fc45cb8bfd7c09d091d54e38e437b727ac1244f1e8d5cb6c

I'd change it (cc @scottn12 )

Comment (tylerbutler) [reply .changeset/seven-ghosts-try.md:3]: Removed the entry for tree.
